PySDM/dynamics/terminal_velocity/power_series.py

```diff
@@ -0,0 +1,27 @@
+"""
+Power series expression
+"""
+import numpy as np
+
+from PySDM.physics import constants as const
+
+
+class PowerSeries:  # pylint: disable=too-few-public-methods
+    def __init__(self, particulator, *, prefactors=None, powers=None):
+        si = const.si
+        self.particulator = particulator
+        self.prefactors = prefactors or [2.0e-1 * si.m / si.s / np.sqrt(si.m)]
```

Comment:
this single-element list value seems to cause Numba deprecation warnings?

Reply:
Yes, this is a warning I've been encountering as well, and something I'm unsure how to mitigate...

Reply:
Okay, I've recreated them as numpy arrays instead.